ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -248,6 +248,10 @@ public void requestBannerAd(@NonNull Context context,
       }
       return;
     }
+
```

Comment:
Hi @jiancchen seems missing call updateCoppaStatus() method in the requestInterstitialAd().

Reply:
@HuFangshuai agreed will need to update